src/hotspot/share/code/nmethod.cpp

```diff
@@ -1587,7 +2161,7 @@ void nmethod::post_compiled_method_unload() {
   assert(_method != nullptr, "just checking");
   DTRACE_METHOD_UNLOAD_PROBE(method());
 
-  // If a JVMTI agent has enabled the CompiledMethodUnload event then
+  // If a JVMTI agent has enabled the nmethod Unload event then
```

Comment:
I think the event is still called CompiledMethodUnload, so this line should probably be reverted.